src/hydration.js

```diff
@@ -0,0 +1,36 @@
+class Hydration {
+  constructor(hydrationData, id) {
+    this.data = hydrationData;
+    this.id = id;
+    this.user = this.findUserHydroData();
+  };
+
+  findUserHydroData() {
```

Comment:
This method finds the data for the user ID correlated with the user ID parameter that gets passed in.  Then we know which user we will be interacting with for the other methods.

Reply:
correct. I know this might be a little unorthodox but it works for what we are trying to accomplish.